src/visu_config.xsd

```diff
@@ -988,12 +1003,42 @@
       <xsd:element name="address" type="address" minOccurs="1" maxOccurs="unbounded"/>
     </xsd:sequence>
-    <xsd:attribute name="upvalue" type="xsd:string" use="optional" />
-    <xsd:attribute name="shortupvalue" type="xsd:string" use="optional" />
-    <xsd:attribute name="downvalue" type="xsd:string" use="optional" />
-    <xsd:attribute name="shortdownvalue" type="xsd:string" use="optional" />
-    <xsd:attribute name="uplabel" type="xsd:string" use="optional" />
-    <xsd:attribute name="downlabel" type="xsd:string" use="optional" />
+    <xsd:attribute name="upvalue" type="xsd:string" use="optional">
+      <xsd:annotation>
+      	  <xsd:documentation xml:lang="en">Defines the value which is sent on long push of the button.</xsd:documentation>
+      	  <xsd:documentation xml:lang="de">Legt fest welcher Wert bei einem langen Tastendruck gesendet wird.</xsd:documentation>
+      </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="shortupvalue" type="xsd:string" use="optional">
+      <xsd:annotation>
+      	  <xsd:documentation xml:lang="en">Defines the value which is sent on short push of the button.</xsd:documentation>
+      	  <xsd:documentation xml:lang="de">Legt fest welcher Wert bei einem kurzen Tastendruck gesendet wird.</xsd:documentation>
+      </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="downvalue" type="xsd:string" use="optional">
+      <xsd:annotation>
+      	  <xsd:documentation xml:lang="en">Defines the value which is sent when releasing the button after a long push.</xsd:documentation>
+      	  <xsd:documentation xml:lang="de">Legt fest welcher Wert beim Loslassen nach einem langen Tastendruck gesendet wird.</xsd:documentation>
+      </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="shortdownvalue" type="xsd:string" use="optional">
+      <xsd:annotation>
+      	  <xsd:documentation xml:lang="en">Defines the value which is sent when releasing the button after a short push.</xsd:documentation>
+      	  <xsd:documentation xml:lang="de">Legt fest welcher Wert beim Loslassen nach einem kurzen Tastendruck gesendet wird.</xsd:documentation>
+      </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="uplabel" type="xsd:string" use="optional">
+      <xsd:annotation>
+      	  <xsd:documentation xml:lang="en">Defines the label which is displayed on the left button.</xsd:documentation>
+      	  <xsd:documentation xml:lang="de">Legt die Beschriftung der linken Taste fest.</xsd:documentation>
+      </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="downlabel" type="xsd:string" use="optional">
+      <xsd:annotation>
+      	  <xsd:documentation xml:lang="en">Defines the label which is displayed on the right button.</xsd:documentation>
+      	  <xsd:documentation xml:lang="de">Legt die Beschriftung der rechten Taste fest.</xsd:documentation>
+      </xsd:annotation>
+    </xsd:attribute>
```

Comment:
not sure if I understood you correctly. The multitrigger has 4 buttons, so there is a top left, top right, bottom left and bottom right button.
I think I did not any button up and/or down-button?